pkgs/games/infra-arcana/release-date.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/include/global.hpp b/include/global.hpp
+index ca476d82..24d0fee8 100644
+--- a/include/global.hpp
++++ b/include/global.hpp
+@@ -13,7 +13,7 @@ const size_t player_name_max_len = 14;
+ 
+ const std::string version_str = "v19.2";
+ 
+-const std::string date_str = __DATE__;
++const std::string date_str = "Nov 6 2017";
+ 
+ const std::string tiles_img_name = "res/images/gfx_16x24.png";
+ const std::string logo_img_name = "res/images/main_menu_logo.png";
```

Comment:
consider to fix `__DATE__` with something like a "substituteInPlace --replace" in a postPatch phase would avoid such patch.

Reply:
I feel that the patch gives better context and avoids ambiguity. I know if I was reading this derivation for the first time I would find the patch easier to grok than a `substituteInPlace` call.

It also avoids potential landmines if future versions of the source add more `__DATE__`s.

I'm happy to switch it to `substituteInPlace` if you want, but I do think the patch is the better option here. Thoughts?

Reply:
no more landmine than a broken patch. And if i don't mess, i think you should not trick with others date but put it to 0. Btw, on tip, author has already moves this predefined variable to another file  src/version.cpp , makes your patch de facto outdated for next release. Would be easier to fix a substituteInPlace with this new file than remove and redo a patch ;)

Reply:
Fair enough. Updated to use `substituteInPlace`.

The only place the date is shown is on the main menu alongside the version number, so I think it makes more sense to use the release date rather than 0.